md/specify-cpu-cores.md

```diff
@@ -0,0 +1,30 @@
+# Custom Tomcat / WildFly start-up script
+
+If you have a Subscription edition license covering fewer CPU cores than are available on your server, you must limit the number of CPUs available.
```

Comment:
I think we should remove the `are` or change it to `those` but I'm definitely not so sure